doc/classes/VisualShaderNodeIntFunc.xml

```diff
@@ -15,16 +15,16 @@
 	</members>
 	<constants>
 		<constant name="FUNC_ABS" value="0" enum="Function">
-			Returns the absolute value of the parameter. Translates to [code]abs(x)[/code] in the Godot Shader Language.
+			Returns the absolute value of the parameter. Translates to [code]abs(x)[/code] in GDShader.
```

Comment:
I think this is an overall great change. The only thing sticking out to me is the grammar of this feels a bit off. But I'm not sure that we can do anything about it. 

Here, it really feels like it should say "in GDShaders" instead of GDShader. However, that doesn't make sense if "GDShader" is the name of the language (i.e. you would never say "in GLSLs"). 

The awkwardness comes from the shading _language_ calling itself a shader. When a shader is actually a program written in the shading language. 

What do people think about distinguishing between GDShaders and the GDShader language? 

In cases like these we could easily refer to either:
```
Returns the absolute value of the parameter. Translates to [code]abs(x)[/code] in GDShaders.
```
or
```
Returns the absolute value of the parameter. Translates to [code]abs(x)[/code] in the GDShader language.
```

Reply:
"in the GDShader language" sounds good to me. There may be some ambiguity with "in GDShaders" because there's no **GDShader** resource, it's just **Shader**

Reply:
Good point. We would have to establish a convention of referring to shaders written in the GDShader language as GDShaders.

Reply:
Suggestion addressed